images/cloud/upgrade.svg

```diff
@@ -0,0 +1,30 @@
+<svg width="267" height="163" viewBox="0 0 267 163" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this the same one currently used in the user limit modal? It seems we are adding it again